File: app/src/main/java/com/norvera/confession/data/models/ExaminationActiveEntry.java
package com.norvera.confession.data.models;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "SIN_ACTIVE")
public class ExaminationActiveEntry {

    @PrimaryKey
    @ColumnInfo(name = "_id")
    private long id;

    @ColumnInfo(name = "DESCRIPTION")
    private String description;

    public ExaminationActiveEntry(long id, String description) {
        this.id = id;
        this.description = description;
    }

    public long getId() {
        return id;
    }

    public String getDescription() {
        return description;
    }

    public void setId(long id) {
        this.id = id;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: app/src/main/java/com/norvera/confession/data/models/ExaminationActiveEntry.kt
package com.norvera.confession.data.models

import android.os.Parcelable
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.android.parcel.Parcelize

@Entity(tableName = "SIN_ACTIVE")
@Parcelize
data class ExaminationActiveEntry(
    @field:PrimaryKey
    @field:ColumnInfo(name = "_id")
    var id: Long,
    @field:ColumnInfo(name = "DESCRIPTION")
    var description: String
) : Parcelable
